ppr-ui/src/store/mutations/mutations-model.ts

```diff
@@ -353,22 +354,10 @@ export const mutateUnsavedChanges = (state: StateIF, unsavedChanges: Boolean) =>
 }
 
 // MHR Registration
-export const mutateMhrRegistrationManufacturerName = (state: StateIF, name: string) => {
-  state.stateModel.mhrRegistration.description.manufacturer = name
-}
-
-export const mutateMhrRegistrationYearOfManufacture = (state: StateIF, year: number) => {
-  state.stateModel.mhrRegistration.description.baseInformation.year = year
-}
-
-export const mutateMhrRegistrationHomeMake = (state: StateIF, make: string) => {
-  state.stateModel.mhrRegistration.description.baseInformation.make = make
-}
-
-export const mutateMhrRegistrationHomeModel = (state: StateIF, model: string) => {
-  state.stateModel.mhrRegistration.description.baseInformation.model = model
-}
-
 export const mutateMhrHomeDescription = (state: StateIF, { key, value }) => {
   state.stateModel.mhrRegistration.description[key] = value
 }
+
+export const mutateMhrBaseInformation = (state: StateIF, { key, value }) => {
+  set(state.stateModel.mhrRegistration.description.baseInformation[key], value)
```

Comment:
is `set` still required here? 
I don't believe it should be

Reply:
Removed set().